avaframe/in3Utils/fileHandlerUtils.py

```diff
@@ -464,6 +464,36 @@ def exportcom1DFAOrigOutput(avaDir, cfg='', addTSteps=False):
     shutil.copy2(pathlib.Path('%s' % inputDir, 'ExpLog.txt'), outDir)
 
 
+def fetchFlowFields(flowFieldsDir, suffix=''):
```

Comment:
is readFields not doing the same?

Reply:
readFields really reads the data of the ascii files, here I just want to have the list of file paths - I moved it to fileHandlerUtils